inception-project/inception
# #867 - Group search results by feature value

**What's in the PR**
* Give the user the option to group the search results by feature value

**How to test manually**
* Go to the annotation page
* Search for something in the search-sidebar and group by feature value
* See if grouping works properly 

**Automatic testing**

**Documentation**
